devices/Sim7080/IoT.Device.Sim7080.nfproj

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <NanoFrameworkProjectSystemPath>$(MSBuildExtensionsPath)\nanoFramework\v1.0\</NanoFrameworkProjectSystemPath>
+  </PropertyGroup>
+  <Import Project="$(NanoFrameworkProjectSystemPath)NFProjectSystem.Default.props" Condition="Exists('$(NanoFrameworkProjectSystemPath)NFProjectSystem.Default.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectTypeGuids>{11A8DD76-328B-46DF-9F39-F559912D0360};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <ProjectGuid>2a19e9d8-d1d2-4999-b32e-d90ff6e6d751</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <FileAlignment>512</FileAlignment>
+    <RootNamespace>IoT.Device.Sim7080</RootNamespace>
+    <AssemblyName>IoT.Device.Sim7080</AssemblyName>
+    <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
```

Comment:
Missing the property to generate Intellisense documentation:
```suggestion
    <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
    <DocumentationFile>bin\$(Configuration)\IoT.Device.Sim7080.xml</DocumentationFile>
```

Reply:
Added the property